docs/configuration/sessions.md

```diff
@@ -32,3 +33,26 @@ The following should be known when using this implementation:
 - Since multiple requests can be made concurrently to the OAuth2 Proxy, this session implementation
 cannot lock sessions and while updating and refreshing sessions, there can be conflicts which force
 users to re-authenticate
+
+
+### Redis Storage
```

Comment:
Might be worth adding here the relevant specific flags required to use the Redis store, so `--redis-connection-url` and `--session-store-type=redis` I guess?

Reply:
I added a usage section below, which includes additional information about supporting Redis sentinel